tracer/src/Datadog.Trace/Debugger/Symbols/ISymbolSink.cs

```diff
@@ -0,0 +1,15 @@
+// <copyright file="ISymbolSink.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Threading.Tasks;
+
+namespace Datadog.Trace.Debugger.Symbols
+{
+    internal interface ISymbolSink : IDisposable
+    {
+        Task StartExtractingAssemblySymbolsAsync();
```

Comment:
nitpick/design quip - all the other "sinks" we use in `LiveDebugger`'s sole responsibility is indeed to be a "sink" in the sense that they receive incoming data and funnel it off to `datadog-agent` (while taking care of whatever subtleties there might be necessary, e.g. in re-transmitting probe statuses, etc etc).

This seems to break that abstraction somehow in that here we have a "sink" that is actually responsible for kicking off the extraction?